controllers/providers/aws/aws.go

```diff
@@ -68,7 +73,7 @@ type AwsWorker struct {
 }
 
 var (
-	DefaultInstanceProfilePropagationDelay = time.Second * 25
+	DefaultInstanceProfilePropagationDelay = time.Second * 35
```

Comment:
I noticed a bug where there is a race condition if IAM doesn't fully propagate nodes will not come up.
This is a real pain, because there is no way around this other than blindly waiting, since the IAM role is 'created' but it still needs some time to propagate before it's used to create a scaling group.
IAM APIs are super old, and some aspects of it are really annoying.

This extra 10 seconds seems to resolve the issue.
This is only an issue when you don't provide your own IAM role and expect the controller to create it